.mailmap

```diff
@@ -553,6 +553,8 @@ Robin H. Johnson <robin.johnson@dreamhost.com>
 Robin Tang <robintang974@gmail.com>
 Roi Dayan <roid@mellanox.com> <roi.dayan@gmail.com>
 Ron Allred <rallred@itrefined.com> <ron-github@neversleep.net>
+Rongqi Sun <sunrongqi@huawei.com> Svelar <sunrongqi@huawei.com>
+Rongqi Sun <sunrongqi@huawei.com> Rongqi Sun (Svelar) <sunrongqi@huawei.co>
```

Comment:
You could only keep `Rongqi Sun <sunrongqi@huawei.com> Rongqi Sun (Svelar) <sunrongqi@huawei.com>` if you want.(Miss a 'm' in <sunrongqi@huawei.co>) Thanks!

Reply:
thanks @Svelar. there was one merge commit 72fe848387dfe085d86269f966ac22e85d9438ae that misspelled your email address that way. i added this mapping so that review still gets credited under your normal name/address